src/Runner.Common/JobServerQueue.cs

```diff
@@ -683,6 +814,19 @@ internal class UploadFileInfo
         public bool DeleteSource { get; set; }
     }
 
+    internal class SummaryUploadFileInfo
+    {
+        public Guid TimelineId { get; set; }
+        public Guid TimelineRecordId { get; set; }
```

Comment:
I still don't get why we need these fields.
```
await _jobServer.CreateStepSymmaryAsync(file.PlanId, file.JobId, file.StepId, file.Path, cancellationTokenSource.Token);
```

Do we only pass PlanId/JobId/StepId/Path to the `_jobServer`?